stemcell_builder/stages/bosh_azure_agent_settings/apply.sh

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env bash
+
+base_dir=$(readlink -nf $(dirname $0)/../..)
+source $base_dir/lib/prelude_apply.bash
+
+# Set SettingsPath but never use it because file_meta_service is avaliable only when the settings file exists.
+cat > $chroot/var/vcap/bosh/agent.json <<JSON
+{
+  "Platform": {
+    "Linux": {
+      "DevicePathResolutionType": "virtio"
+    }
+  },
+  "Infrastructure": {
+    "NetworkingType": "dhcp",
```

Comment:
It is because that Windows Azure Linux agent needs to access network before generating CustomData. bosh-agent will read CustomData before configuring network. So we have to set this.